tests/notifications/test_creator_notifications.py

```diff
@@ -87,3 +87,53 @@ def test_notify_creator_on_moderator_feedback(proposal_factory, client):
     assert mail.outbox[1].to[0] == creator.email
     assert mail.outbox[1].subject.startswith(
         'Feedback for your contribution')
+
+
+@pytest.mark.django_db
+def test_notify_comment_creator_on_blocked(idea, comment_factory,
+                                           apiclient, admin):
+    """Check if creator gets email on comment blocked."""
+    comment = comment_factory(content_object=idea)
+    creator = comment.creator
+    # FIXME: use moderator instead of admin when rules are there
+    # moderator = comment.project.moderators.first()
+
+    # 3 emails because of creator notification for reaction on idea
+    assert len(mail.outbox) == 3
+
+    url = reverse(
+        'moderationcomments-detail',
+        kwargs={
+            'project_pk': comment.project.pk,
+            'pk': comment.pk,
+        }
+    )
+    data = {
+        "is_blocked": True,
+    }
+
+    # apiclient.force_authenticate(user=admin)
+    # response = apiclient.patch(url, data, format='json')
```

Comment:
Here, I get a 404, and I don't get that! @Rineee do you have an idea? But anyway, that is a problem for Monday.

Reply:
Ah! Thank you! That makes total sense, also the 404 then. :) I will try that!